firebase.json

```diff
@@ -0,0 +1,42 @@
+{
+  "hosting": {
+    "public": "webplugin/build",
+    "ignore": [
+      "firebase.json",
+      "**/.*",
+      "**/node_modules/**"
+    ],
+    "headers": [
+      {
+        "source": "**/*.html",
+        "headers": [
+          {
+            "key": "Access-Control-Allow-Origin",
+            "value": "*"
+          }
+        ]
+      },
```

Comment:
There is one file, `kommunicate.app`, which is the initial point of the script basically, we are giving that URL in the customer script, and it does not follow any file convention these regex will not match that file

Reply:
added